components/ErrorNotification.tsx

```diff
@@ -111,20 +111,23 @@ const ErrorTitle = styled('h1')`
 
 const ErrorNotification = ({
```

Comment:
Reusing this component for validation errors in the modal. Had to update certain aspects of it for additional reusability.